Voice/Weather Phone IVR - Node.js/index.js

```diff
@@ -0,0 +1,175 @@
+require("dotenv").config();
+const parsePhoneNumber = require("libphonenumber-js");
+const axios = require("axios");
+let sw = require("@signalwire/realtime-api");
+let { Voice, Messaging } = sw;
+
+const DC_WEATHER_PHONE = "+12025891212";
+const PHONE_NUMBER = process.env.PHONE_NUMBER;
+
+const realtimeConfig = {
+  project: process.env.PROJECT_ID,
+  token: process.env.API_TOKEN,
+  contexts: ["office"],
+};
+
+const messageClient = new Messaging.Client(realtimeConfig);
+const voiceClient = new Voice.Client(realtimeConfig);
+
+console.log("Waiting for calls...");
+voiceClient.on("call.received", async (call) => {
+  console.log("Got a call from", call.from, "to number", call.to);
+  await call.answer();
+  console.log("Inbound call answered");
+  try {
+    let welcome = await call.playTTS({
+      text: "Hello! Welcome to Knee Rub's Weather Helpline.",
+      gender: "male",
+    });
+    await welcome.waitForEnded();
+    console.log("Welcome text said");
+
+    let digits, terminator, type;
+    const cmdPrompt = await call.promptTTS({
+      text: "Please enter 1 for Washington weather, 2 for washington weather message, 3 to play rain dance, 4 to send rain dance",
+      digits: {
+        max: 1,
+        digitTimeout: 15,
+      },
+    });
+    let cmdResult = await cmdPrompt.waitForResult();
+    type = cmdResult.type;
+    digits = cmdResult.digits;
+    terminator = cmdResult.terminator;
+    console.log(
+      "Prompted for digits, received digits",
+      type,
+      digits,
+      terminator
+    );
+
+    if (digits === "1") {
+      await call.connectPhone({
+        from: call.from,
+        to: DC_WEATHER_PHONE,
+        timeout: 30,
+        ringback: new Voice.Playlist().add(
+          Voice.Playlist.TTS({
+            text: "ring. ring. ring. ring. ring. ring. ring. ring",
```

Comment:
I think this has an awesome comedic effect, but couldn't we use Ringtone instead?

Reply:
I just left it there because it was funny, tbh. Should I switch back to the standard ringtone?